deploy/cr.yaml

```diff
@@ -53,7 +53,17 @@ spec:
 #        cpu: 500m
 #        memory: "256Mi"
 #    affinity:
-#      antiAffinityType: preferred
```

Comment:
@cap1984 should we remove it? I think no.

Reply:
I think we need to keep it for backward compatibility.